src/main/common/utils.h

```diff
@@ -118,3 +118,5 @@ void * memcpy_fn ( void * destination, const void * source, size_t num ) asm("me
 #else
 #define FALLTHROUGH do {} while(0)
 #endif
+
+#define ALIGNED(x) __attribute__ ((aligned(x)))
```

Comment:
Is this actually used anywhere?

Reply:
My mistake. I left it over while porting the driver from INAV.